### ISSUE TITLE
Incompatible with the mod Palladium

### ISSUE THREAD

**Issue by NavalAce** [ItsPasi/natural-motionblur-fabric#5]
Screenshot:
![can't see](https://github.com/user-attachments/assets/f207d268-4fa8-42f0-8def-fc48f7e193ff)
Log:
https://mclo.gs/7i7jVPh
Also I'm able to see if I enable shaders, because they disable motion blur

**Comment by ItsPasi** [comment]
Can you test without other mods?

**Comment by NavalAce** [comment]
Mod list from the log file. Note that disabled mods don't affect anything
 [✔] accurate-block-placement-1.2.5
  [✔] animatica-0.6.1 1.21
  [✔] appleskin-fabric-mc1.21-3.0.5
  [✔] architectury-13.0.6-fabric
  [✔] authme-8.0.0 1.21
  [✔] autohud-7.2.7 1.21-fabric
  [✔] BadOptimizations-2.1.4-1.21
  [✔] BetterF3-11.0.2-Fabric-1.21.1
  [✔] bettermounthud-1.2.4
  [✔] capes-1.5.4 1.21-fabric
  [✔] cardinalboats-1.4.2
  [✔] caxton-0.6.0-alpha.2 1.21.1-FABRIC
  [✔] chattools-1.21.1-v2.3.4
  [✔] chesttracker-2.4.13 1.21.1
  [✔] Chunky-1.4.16
  [✔] citresewn-1.2.1 1.21
  [✔] client_maps-1.0.5
  [✔] ClientSort-Fabric-1.21-0.9.0
  [✔] cloth-config-15.0.130-fabric
  [✔] collective-1.21.1-7.84
  [✔] continuity-3.0.0-beta.5 1.21
  [✘] Controlify-2.0.0-beta.14 1.21-fabric.jar (disabled)
  [✔] Controlling-fabric-1.21.1-19.0.3
  [✔] cubes-without-borders-2.1.1 1.21
  [✔] Debugify-1.21.1 1.0
  [✔] DistantHorizons-2.2.1-a-1.21.1-neo-fabric
  [✔] DistinguishedPotions-v21.1.0-1.21.1-Fabric
  [✔] dynamiccrosshair-8.1 1.21-fabric
  [✔] dynamic-fps-3.6.3 minecraft-1.21.0-fabric
  [✔] dynamic-fullbright-1.2.0 1.21
  [✘] e4mc_minecraft-5.1.0.jar (disabled)
  [✔] elytrapitch-1.3.1
  [✔] elytratime-2.1.0
  [✔] emi_loot-0.7.4 1.21 fabric
  [✔] emi-1.1.12 1.21 fabric
  [✔] entity_model_features_fabric_1.21-2.2.3
  [✔] entity_texture_features_fabric_1.21-6.2.2
  [✔] entityculling-fabric-1.7.0-mc1.21
  [✔] ep-msg-encryption-2.0.0
  [✘] EssentialClient-1.5.2 mc1.21.jar (disabled)
  [✔] fabric-api-0.103.0 1.21.1
  [✘] fabric-carpet-1.21-1.4.147 v240613.jar (disabled)
  [✔] fabric-elytra-autopilot-1.21.1-3.0.1
  [✔] fabric-language-kotlin-1.12.1 kotlin.2.0.20
  [✔] fabricskyboxes-0.7.4 mc1.21
  [✔] fabrishot-1.14.1
  [✔] fasterrandom-5.1.0
  [✔] fastquit-3.0.0 1.20.6
  [✔] ferritecore-7.0.0-fabric
  [✘] flashback-0.7.0.jar (disabled)
  [✔] ForgeConfigAPIPort-v21.1.1-1.21.1-Fabric
  [✔] fsb-interop-1.4.0 mc1.21-build.54
  [✔] fzzy_config-0.5.1 1.21
  [✔] ImmediatelyFast-Fabric-1.2.21 1.21.1
  [✔] indium-1.0.35 mc1.21
  [✔] inventive_inventory-1.0.1
  [✔] InventoryProfilesNext-fabric-1.21-2.0.4
  [✔] iris-fabric-1.8.0-beta.3 mc1.21.1
  [✔] itemscroller-fabric-1.21.1-0.24.50
  [✘] jei-1.21.1-fabric-19.9.0.121.jar (disabled)
  [✔] JJElytraSwap-2.01 (2)
  [✔] keybind_fix-1.0.1
  [✔] language-reload-1.6.1 1.21
  [✔] libIPN-fabric-1.21-6.0.2
  [✔] libjf-3.17.0
  [✔] litematica-fabric-1.21.1-0.19.50
  [✔] lithium-fabric-mc1.21.1-0.13.0
  [✔] malilib-fabric-1.21.1-0.21.0
  [✔] map-in-slot-3.1.2
  [✔] memorysweep-1.21.1-1.1.1-fabric
  [✔] minihud-fabric-1.21.1-0.32.50
  [✔] mixintrace-1.1.1 1.17
  [✔] modelfix-1.21-1.6-fabric
  [✔] modernfix-fabric-5.19.3 mc1.21.1
  [✔] modmenu-11.0.2
  [✔] moreculling-1.21-0.27.1
  [✔] MouseTweaks-fabric-mc1.21-2.26
  [✔] naturalmotionblur-1.1.2 mc1.21
  [✘] NoChatReports-FABRIC-1.21-v2.8.0.jar (disabled)
  [✔] noisium-fabric-2.3.0 mc1.21-1.21.1
  [✔] null-bind-1.0.2
  [✔] optigui-2.3.0-beta.2 1.21
  [✔] overworld_coordinates-2.2.0
  [✔] paginatedadvancements-2.4.2
  [✔] Palladium-1.21x-1.0.9
  [✔] plasmovoice-fabric-1.21-2.0.10
  [✔] polytone-1.21-2.3.8-fabric
  [✔] puzzle-1.6.3 1.21
  [✔] PuzzlesLib-v21.1.18-1.21.1-Fabric
  [✘] reeses-sodium-options-fabric-1.8.0-beta.2 mc1.21.1.jar (disabled)
  [✔] remote_player_waypoints_for_xaero-fabric-3.2.0-1.21-1.21.1
  [✔] replaymod-1.21-2.6.19
  [✔] Resourcify (1.21 -fabric)-1.4.1
  [✔] respackopts-4.10.0
  [✔] rrls-5.0.7 mc1.21-fabric
  [✔] screenshot-to-clipboard-fabric-1.0.10
  [✔] Searchables-fabric-1.21.1-1.0.1
  [✔] slyde-1.7.5
  [✔] sodium-extra-fabric-0.6.0-beta.1 mc1.21.1
  [✔] sodium-fabric-0.6.0-beta.1 mc1.21
  [✔] sound-physics-remastered-fabric-1.21.1-1.4.6
  [✔] stack-to-nearby-chests-mc1.21-0.5.8
  [✔] tnt-timer-2
  [✔] ToadLib-1.21x-1.1.6
  [✔] tweakermore-v3.21.0-mc1.21.1
  [✔] tweakeroo-fabric-1.21.1-0.21.50
  [✔] videotape-1.3.0
  [✔] waveycapes-fabric-1.4.9-mc1.21
  [✔] world-host-0.4.14 1.21.1-fabric
  [✔] XaeroPlus-2.24.3 fabric-1.21-WM1.39.0-MM24.5.0
  [✔] Xaeros_Minimap_24.5.0_Fabric_1.21
  [✔] xaeros-map-chest-tracker-integration-mc1.21.1-v3.0.2
  [✔] XaerosWorldMap_1.39.0_Fabric_1.21
  [✔] YetAnotherConfigLib-3.5.0 1.21-fabric
  [✔] yosbr-0.1.2
  [✔] Zoomify-2.14.0 1.21

**Comment by ItsPasi** [comment]
Yeah I just saw it I am just stupid lol. Do you have a modpack you are playing with so I can easily test or maybe you can find out yourself whih mod is causing it / test without mods first.

**Comment by NavalAce** [comment]
My modpack is Fabulously Optimized as base, however I added bunch of mods on top of it. You could try base FO to see if it works first

https://modrinth.com/modpack/fabulously-optimized

**Comment by ItsPasi** [comment]
Alright the issue is the mod "Palladium". I am gonna see if I can make it compatible but since that is an optimization mod that probably plays with the rendering engine I am not sure if I an fix it.

**Comment by ItsPasi** [comment]
Yeah not really much I can do I think. The mod changes the rendering of on screen effects. I found out just spectating mobs with on-screen effects also breaks that mod. I'll put the mod to the list of incompatible mods for now.

**Comment by NavalAce** [comment]
Okay, I'll stop using Palladium for now. Should I open a new issue on Palladium devs GitHub then?

**Comment by ItsPasi** [comment]
No there is already an issue related to blackscreens and I commented in there.

**Comment by NavalAce** [comment]
Okay, do we close this issue for now?

**Comment by ItsPasi** [comment]
I'll keep it open until there is feedback from the dev to see if it's fixable.

**Comment by ItsPasi** [comment]
Appears to be fixed with newer versions of Palladium
